quesma/model/bucket_aggregations/terms.go

```diff
@@ -56,3 +65,43 @@ func (query Terms) String() string {
 	}
 	return "significant_terms"
 }
+
+func (query Terms) sumDocCounts(rows []model.QueryResultRow) int {
+	sum := 0
+	if len(rows) > 0 {
+		switch query.docCount(rows[0]).(type) {
+		case int64:
+			for _, row := range rows {
+				sum += int(query.docCount(row).(int64))
+			}
+		case uint64:
+			for _, row := range rows {
+				sum += int(query.docCount(row).(uint64))
+			}
+		default:
+			logger.WarnWithCtx(query.ctx).Msgf("unknown type for terms doc_count: %T, value: %v",
+				query.docCount(rows[0]), query.docCount(rows[0]))
+		}
+	}
+	return sum
+}
+
+func (query Terms) docCount(row model.QueryResultRow) any {
+	return row.Cols[len(row.Cols)-1].Value
+}
+
+func (query Terms) key(row model.QueryResultRow) any {
+	return row.Cols[len(row.Cols)-2].Value
+}
+
+func (query Terms) parentCount(row model.QueryResultRow) any {
```

Comment:
I don't like `parent_count`, took me a while before I understood it, why not `total_count`?